db/migrate/20210720075622_create_user_dismissed_introducers.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUserDismissedIntroducers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :user_dismissed_introducers do |t|
+      t.belongs_to :user, null: false, foreign_key: true
+
+      t.string :name, null: false
```

Comment:
At the risk of you hating me, I think we should change this to be `slug` rather than `name`. I think we could use `name` to provide a human readable name for this later, which we show in the settings (to turn introducers back on maybe). And `name` is always human-readable and `slug` is generally this format.

Reply:
No I don't mind. Good argument